src/config.ts

```diff
@@ -741,6 +778,27 @@ export const FF_MIT_VOUCHER_ENABLED =
 export const FF_USER_AGE_LIMIT_ENABLED =
   process.env.FF_USER_AGE_LIMIT_ENABLED === "1";
 
+// IOLOGIN FF variable
+export const FF_IOLOGIN = pipe(
+  process.env.FF_LOGIN,
```

Comment:
```suggestion
  process.env.FF_IOLOGIN,
```

See https://github.com/pagopa/io-infra/pull/556/files

Reply:
solved in https://github.com/pagopa/io-backend/pull/1012/commits/2f27b7e1455672699d54b3b8e5b38750db8bb08f